source/MRViewer/MRRibbonMenu.h

```diff
@@ -213,12 +216,16 @@ class MRVIEWER_CLASS RibbonMenu : public ImGuiMenu
     float openedTimer_{ openedMaxSecs_ };
 
     int activeTabIndex_{ 0 };
-    MenuItemsList quickAccessList_;
     RibbonFontManager fontManager_;
     RibbonButtonDrawer buttonDrawer_;
 
     AsyncTimer asyncTimer_;
     std::thread timerThread_;
+
+    MenuItemsList quickAccessList_;
+    MenuItemsList toolbarListCustomize_;
```

Comment:
please add comment why we need two of them

Reply:
fixed